### ISSUE TITLE
compiling issues :P

### ISSUE THREAD

**Issue by EliezerSolinger** [cjac/jenkara#1]
I tried to compile this software(for fun) but is too old, uses gtk2 stuff dependency and is not easy on modern linux distributions .

**Comment by cjac** [comment]
yeah, I should re-write it with gtk3+ ; we even got the OpenGL operations into the core library

https://docs.gtk.org/gtk3/class.GLArea.html
